test/http.js

```diff
@@ -544,6 +545,36 @@ describe('HTTP APIs', function () {
         .expect('content-type', /text\/turtle/)
         .expect(200, done)
     })
+
+    it('should create a container with a url name', (done) => {
+      let containerName = 'https://example.com/page'
+      let expectedDirName = '/post-tests/https%3A%2F%2Fexample.com%2Fpage/'
+      server.post('/post-tests/')
+        .set('slug', containerName)
+        .set('content-type', 'text/turtle')
+        .set('link', '<http://www.w3.org/ns/ldp#BasicContainer>; rel="type"')
+        .expect(201)
+        .end((err, res) => {
+          if (err) return done(err)
+          try {
+            assert.equal(res.headers.location, expectedDirName,
+              'Uri container names should be encoded')
+            let createdDir = fs.statSync(path.join(__dirname, 'resources', expectedDirName))
+            assert(createdDir.isDirectory(), 'Container should have been created')
+          } catch (err) {
+            return done(err)
+          }
+          done()
+        })
+    })
+
+    it('should be able to access new url-named container', (done) => {
```

Comment:
Do we not clean up after every single test? I would have expected these two tests to be the same test, and for the resource `/post-tests/https%3A%2F%2Fexample.com%2Fpage/` *not* to exist in the scope of other tests in this describe block.